constructor/fcp.py

```diff
@@ -254,13 +256,12 @@ def _solve_precs(name, version, download_dir, platform, channel_urls=(), channel
     # JRG: This only applies to the `base` environment; `extra_envs` are exempt
     if not extra_env:
         specs = (*specs, "python")
-    if verbose:
-        if environment:
-            print(f"specs: <from existing environment '{environment}'>")
-        elif environment_file:
-            print(f"specs: <from environment file '{environment_file}'>")
-        else:
-            print("specs:", specs)
+    if environment:
+        logger.debug(f"specs: <from existing environment '{environment}'>")
+    elif environment_file:
+        logger.debug(f"specs: <from environment file '{environment_file}'>")
```

Comment:
Same comment about %-format

Reply:
Updated the PR description with some reasoning on this topic.